Welolo/client/src/components/TRANSACTION_FORM.js

```diff
@@ -166,115 +163,76 @@ class TRANSACTION_FORM extends Component {
-                            name="sender_quantity"
-                            id="sender_quantity"
-                            value={this.state.message.sender_quantity}
-                            onChange={this.onHandleChange}
-                        />
-                    </div>
-                    <div>
-                        <label htmlFor="body">Message:</label>
-                        <textarea 
-                            name="body"
-                            id="body"
-                            value={this.state.message.body}
-                            onChange={this.onHandleChange}
-                        />
-                    </div>
-                    <button type="submit" htmlFor="submit" name="submit" id="submit" disabled={this.state.submitting}>
-                        Send message
-                    </button>
-                </form>                     
-                <div>{this.state.error ? <p>Please try again! phone number you entered was not a valid phone number</p> : <p></p>}</div>
+  render(){
+    var readOnly;
+    if(this.state.message.sender_quantity === ''){
+      readOnly=false;
+    }
+    return (
+      <div id="payment_wrapper">
+        <h3></h3>
+        <form onSubmit={this.onSubmit}>
+          <div id="payment_form_recipient_name_wrapper">
+            <div className="label_wrapper">
+              <label id="recipient_name_label" htmlFor="recipient_name">Recipient Full Name</label>
             </div>
-        );
-      }
-      else
-      {
-        return (
-          <div>
-              <form onSubmit={this.onSubmit}>
-                  <h1>Payment</h1>
-                  <div>
-                      <label htmlFor="recipient_name">Recipient Name:</label>
-                      <input 
-                          type="text"
-                          name="recipient_name"
-                          id="recipient_name"
-                          value={this.state.message.recipient_name}
-                          onChange={this.onHandleChange}
-                      />
-                  </div>
-                  <div>
-                      <label htmlFor="recipient">Recipient Phone Number (ex: +12223334444):</label>
-                      <input
-                          type="tel"
-                          name="recipient"
-                          id="recipient"
-                          value={this.state.message.recipient}
-                          onChange={this.onHandleChange}
-                      />
-                  </div>
-                  <div>
-                      <label htmlFor="sender_quantity">Quantity (USD):</label>
-                      <input
-                          type="text"
-                          name="sender_quantity"
-                          id="sender_quantity"
-                          value={this.state.message.sender_quantity}
-                          onChange={this.onHandleChange}
-                          readOnly
```

Comment:
This is the else statement that locks the payment text box. Specifically that readOnly is what you'll need. It's kind of messy but it was the best way I could find that did what we wanted it to.